docs/python.md

```diff
@@ -7,7 +7,12 @@ type: markdown
 
 ## Python Developer Guide
 
+The Abseil Python Developer Guide provides programming documentation for the
+Abseil Python code on
+[https://github.com/abseil/abseil-py](https://github.com/abseil/abseil-py). Much
+of the code within that repository is heavily documented. This guide is designed
+to augment those code comments. For full reference documentation, consult the
+source files directly.
+
 * [Python Quickstart](python/quickstart)
-* [Python Programming Guides](python/guides/)
-* [Python Styleguide](python/styleguide)
-* [Python Release Notes](python/releasenotes)
\ No newline at end of file
+* [Python Programming Guides](python/guides/)
```

Comment:
I don't see the landing page in this PR. Did you not add it?

python/guides/index.md

Reply:
Oh, OK. No worries. I thought it might need to be updated but I guess not.